packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -36,42 +39,60 @@ export const generateTestData = async (hostname?: string): Promise<boolean> => {
   return generate();
 };
 
+export const generateTestDataForSlug = async (
+  stdout: NodeJS.WriteStream,
+  slug: string,
+  locale: string,
+  projectStructure: ProjectStructure
+): Promise<any> => {
+  const templateModules = await loadTemplateModules(
+    getTemplateFilePathsFromProjectStructure(projectStructure),
+    true,
+    false
+  );
+  const featuresConfig = await getFeaturesConfig(templateModules);
+  const featuresConfigForEntityPages: FeaturesConfig = {
```

Comment:
nit: can we rename this to indicate this only contain non-static / entity related features?

Reply:
I tried to do that with `featuresConfigForEntityPages` but definitely open to suggestions!